clientv3/retry.go

```diff
@@ -36,6 +36,14 @@ func (c *Client) newRetryWrapper() retryRpcFunc {
 			eErr := rpctypes.Error(err)
 			// always stop retry on etcd errors
 			if _, ok := eErr.(rpctypes.EtcdError); ok {
+				if grpc.Code(err) == codes.Unavailable {
+					// try to request another endpoint at next rpc
+					// leave this fail
+					// already do anti-too-frequently switch
+					// at most switch one time per 30 seconds
```

Comment:
yes, I delete all comments, and rename to TrySwitchEndpoint, that is a better name tells what the function does